feature-libs/cart/base/components/cart-coupon/cart-coupon.component.ts

```diff
@@ -84,8 +89,15 @@ export class CartCouponComponent implements OnInit, OnDestroy {
 
     this.cartVoucherService.resetAddVoucherProcessingState();
 
+    const shouldHaveRequiredValidator = !this.featureConfigService?.isEnabled(
```

Comment:
I think we should have a TODO comment here with the ticket number for breaking change cleanup later.